src/js/ia/IAPage.js

```diff
@@ -155,6 +155,25 @@
                         $(".ia-issues ul li").show();
                     });
 
+		    if(localStorage.dismissInfo) {
```

Comment:
@jagtalon I'm curious why did you choose to use localStorage instead of checking the new_contributor value?